### ISSUE TITLE
Optimize JSON performance via different library

### ISSUE THREAD

**Issue by nvkelso** [tilezen/tilequeue#139]
Profiling tile generation speed with New Relic we've found we spend a good bit of time generating the basic GeoJSON (*.json) tiles – and any TopoJSON (*.topojson) or MVT (*.mvt) tile generated also is required to pass thru the *.json format as a first step.

We think that `import simplejson` in Python (instead of `import json`) will speed things up significantly for all formats. 

Please also include before/after benchmarks.

(Requires additional tileserver work, link to this issue.)

**Comment by rmarianski** [comment]
> and any TopoJSON (.topojson) or MVT (*.mvt) tile generated also is required to pass thru the *.json format as a first step.

To clarify, I don't think it's required, we just do it that way. Circumstances have changed since though, eg we have xonacatl and enqueue the tile coordinate, so I think we should at least re-evaluate whether this is still necessary.

**Comment by nvkelso** [comment]
Let's focus just on `simplejson` configuration change here.

**Comment by iandees** [comment]
I ran some benchmarks this afternoon and came to a surprising finding: Python's built-in `json` is just as fast at what we're doing as `simplejson`. In fact in some cases it's ~1-5% slower. This was surprising to me, but I found [this fairly recent blog post](http://artem.krylysov.com/blog/2015/09/29/benchmark-python-json-libraries/) that benchmark's some Python JSON libraries that gives similar findings.

Their benchmark post includes another, less-featureful library called `ujson`. It gives a fairly sizeable (~10-20%) bump in speed for our topojson and json formats.

My raw numbers are in [this Google Sheet](https://docs.google.com/a/mapzen.com/spreadsheets/d/1RN2ShzRmeV8haCbuvQo9KIrjWf0ePC2TapqWiAhrCiw/edit?usp=sharing) for now, and I'll work on summarizing them in the morning.
